### PR TITLE
Test against php 8.4

### PR DESCRIPTION
**Pull request by sergiy-petrov**

### CODE REVIEW

File: src/PDF.php [hunk 1]
```diff
@@ -193,7 +193,7 @@ public function output(array $options = []): string
     /**
      * Save the PDF to a file
      */
-    public function save(string $filename, string $disk = null): self
+    public function save(string $filename, ?string $disk = null): self
```
Comment (sergiy-petrov) [inline src/PDF.php:196]: image

**Comment by barryvdh** [comment]
Thanks!

**Comment by Marc-Scheidtmann** [comment]
Hello @barryvdh, happy new year. May I ask whether you could be so kind to release this change? Thanks a lot for your work. 👍🏼

**Comment by barryvdh** [comment]
Done

**Pull request merged**